src/types.ts

```diff
@@ -82,6 +82,10 @@ export type Command =
       type: 'returnHtml'
     }
   | {
+      type: 'returnPDF',
+      options: object
```

Comment:
@schickling Take a look at 7db33a1. I'm not very familiar with TypeScript, so feel free to correct any mistakes I may have made.